docs/working-remotely-ssh.md

```diff
@@ -0,0 +1,49 @@
+# Working remotely with Visual Studio Code
+
+It is possible to work remotely on a machine containing the required opensafely software using Visual Studio Code.
+This may be useful in cases where it is not possible to install all of the required sofware to run opensafely on your local machine, or where running this required software severly affects the performance of your local machine. 
+
+This guide walks you through the steps required to set up the machine running the opensafely software (the "server") and the local machine from which you will access it (the "client").
+
+## 1. Server setup
+
+First, ensure that the opensafely software and all of its pre-requisites have been installed using the [Getting Started guide](getting-started). If using this guide to connect to Gitpod, please skip to step 2.
+
+### SSH server installation and configuration
+The steps in this guide require enabling access to the server using secure shell protocol (SSH).
```

Comment:
nit: I think "secure shell protocol" is conventionally capitalised as "Secure Shell Protocol", e.g [see Wikipedia's page](https://en.wikipedia.org/wiki/Secure_Shell_Protocol) and the RFC documents linked from there.